contracts/core/SoundEditionV2_1.sol

```diff
@@ -232,7 +232,7 @@ contract SoundEditionV2_1 is ISoundEditionV2_1, ERC721AQueryableUpgradeable, ERC
         uint8 tier,
         address[] calldata to,
         uint256 quantity
-    ) external payable onlyRolesOrOwner(ADMIN_ROLE) returns (uint256 fromTokenId) {
+    ) external payable onlyRolesOrOwner(ADMIN_ROLE | MINTER_ROLE) returns (uint256 fromTokenId) {
```

Comment:
Checked this in context. The change is to allow `airdrop` to be called by the minter to facilitate the minter's new airdrop functionality. This checks out no response needed